english/change-change/change-change

```diff
@@ -0,0 +1 @@
+-All of a sudden everything stops, and you find yourself in the middle of a clothing store, where all of the merchandise is made with coins. You look down and realize that your clothes are made of coins as well. You decide it's time to change the change on you, and gear up for your next adventure. Time to change change.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.